index.html

```diff
@@ -1057,6 +1057,47 @@ <h2>The <dfn>InteractionOutput</dfn> interface</h2>
     <section><h3>The <dfn>check data schema</dfn> algorithm</h3>
       To run the <a>check data schema</a> steps on |payload| and |schema:object|,
       <ol>
+        <li>
+          If |schema| is `null` or `undefined`, return `undefined`.
+        </li>
+        <li>
+          <!--
+            TODO: Should we just continue with the rest of the algorithm here?
+          -->
+          If |schema|.|default| is not |undefined| and |payload| is undefined,
+          let |payload| be the value of |schema|.|default|.
+        </li>
+        <li>
+          Let |const| be |schema|.|const|.
+          If |const| is not |undefined|,
+          <ol>
+            <li>
+              <!-- TODO: Should we already return here? -->
+              If |const| equals |payload|, return |payload|.
+            </li>
+            <li>
+              If |const| does not equal |payload|, throw {{TypeError}} and stop.
+            </li>
+          </ol>
+        </li>
+        <li>
+          If |schema|.|enum| is not |undefined| and none of the elements in
+          |schema|.|enum| equal |payload|, throw {{TypeError}} and stop.
+        </li>
+        <li>
+          Let |oneOf| be |schema|.|oneOf|.
+          If |oneOf| is not |undefined|,
+          <ol>
+            <li>
+              For each |subSchema| in |oneOf|, run the <a>check data schema</a>
+              steps on |payload| and |subSchema|.
+            </li>
+            <li>
+              If none or more than two of these runs do not throw, throw
+              {{TypeError}} and stop.
+            </li>
```

Comment:
I now adjusted this step of the algorithm as can be seen here. As this term checks whether there is exactly one matching `subSchema`, 0 or > 2 successful runs would indicate an invalid payload here.

Reply:
(Just realized that this needs to be 0 or > 1 run instead, updated in 2a3365f0e1e1d1586d289287a1862d2b72bc4dd8.